src/sentry/event_manager.py

```diff
@@ -664,17 +662,6 @@ def to_values(v):
             del data['stacktrace']
             # TODO(ja): Remove meta data of data['stacktrace'] here, too
 
-        # Exception mechanism needs SDK information to resolve proper names in
```

Comment:
Not quiet sure why this was removed.

Reply:
I was removing that method `get_sdk_from_event` which is not used in our code.